psydac/linalg/stencil.py

```diff
@@ -51,18 +60,24 @@ def __init__( self, *args, **kwargs ):
             self._init_serial  ( *args, **kwargs )
 
     # ...
-    def _init_serial( self, npts, pads, periods, dtype=float ):
+    def _init_serial( self, npts, pads, periods, multiplicity=None, dtype=float ):
+
+        if multiplicity is None:
+            multiplicity = tuple(1 for i in pads)
```

Comment:
we need this information in the `StencilVectorSpace` because we need to set it for the cart in the paralle case, also the `StencilVector` and `StencilMatrix` use  this information.

Reply:
done for the naming

Reply:
This is good enough for now. We'll improve in a future refactoring of Psydac